frontend/src/components/elements/Table/styled-components.ts

```diff
@@ -29,6 +29,7 @@ export const StyledTableContainer = styled.div(({ theme }) => ({
 export const StyledTable = styled.table(({ theme }) => ({
   width: theme.sizes.full,
   marginBottom: theme.spacing.lg,
+  marginTop: theme.spacing.lg,
```

Comment:
What is the reason why some elements use `theme.spacing.lg` and others use `theme.spacing.xl`. Is there sometimes existing spacing?

Reply:
Yeah, exactly. Tables already have some padding around them (see screenshot below) so the smaller `marginTop` tries to accommodate for that

<img width="742" alt="Screen Shot 2022-08-01 at 10 17 49 AM" src="https://user-images.githubusercontent.com/34423371/182156778-590ffba0-34a9-46fd-8001-76fca8166ed4.png">